### ISSUE TITLE
Forgot password location for sign in template

### ISSUE THREAD

**Issue by Wolfr** [smartcoop/design#166]
https://design.smart.coop/development/template-examples/sign-in-form.html

On this sign in template I feel the “forgot password has a bit of a weird location. Why don't we put it outside of the panel, centered?

**Comment by thusc** [comment]
I agree. Also, sometimes, that kinf of screen has additional link. Maybe to a contact page or something. Maybe we should make sure we can accomodate that ?

For instance here, we have additional links (granted, it's maybe not the best way to organize stuff, but it exists): https://account.ubik.be/accountlogin.aspx

**Comment by Wolfr** [comment]
> I agree. Also, sometimes, that kinf of screen has additional link. Maybe to a contact page or something. Maybe we should make sure we can accomodate that ?
> 
> For instance here, we have additional links (granted, it's maybe not the best way to organize stuff, but it exists): https://account.ubik.be/accountlogin.aspx

Nothing stops us from having more of marketing style header/footer on these pages where we can have additional links. I'll try to find/design a good example.
